test/console/graphql/queries/deployments/cluster_queries_test.exs

```diff
@@ -583,6 +583,7 @@ defmodule Console.GraphQl.Deployments.ClusterQueriesTest do
       admin = admin_user()
       insert_list(2, :cluster, current_version: "1.29", pinged_at: Timex.now(), upgrade_plan: %{compatibilities: true, incompatibilities: true, deprecations: true})
       insert_list(3, :cluster, current_version: "1.30", pinged_at: Timex.now() |> Timex.shift(days: -1))
+      insert_list(2, :cluster, current_version: "1.31", pinged_at: Timex.now() |> Timex.shift(days: -1))
```

Comment:
the reason you had to change the asserts is you didn't modify the top set of clusters to `1.30` and the following to `1.31` to preserve the version pattern

Reply:
this looks better: https://github.com/pluralsh/console/pull/1524/files#diff-9356619c481fa6b7722ae779fe4a8d30d3038acfe35db254f7db13039f6e0cee